solutions/observability/get-started/quickstart-monitor-hosts-with-opentelemetry.md

```diff
@@ -86,9 +86,9 @@ Follow these steps to collect logs and metrics using the EDOT Collector:
 Logs are collected from setup onward, so you won’t see logs that occurred before starting the EDOT Collector.
 ::::
 
-:::
+::::
 
-:::{tab-item} Serverless
+::::{tab-item} Serverless
```

Comment:
Looks like the tabs are getting messed up here. I've attached a screenshot. I think the last set of colons in line 115 would need to be 5 colons, and line 113 would need to be 4 colons. Was it not formatting correctly before?

![image](https://github.com/user-attachments/assets/6a494b6e-004b-462d-90e2-7688e53b72d9)

Reply:
You are correct! Thx